plugin/src/main/kotlin/com/slapin/gsp/task/PublishToGalaxyStore.kt

```diff
@@ -111,13 +118,20 @@ constructor(
     withRetry(
       delay = Duration.ofSeconds(10),
       count = 5,
-      onFailure = { logger.error("Failed to register new binary") }
+      onFailure = { error("Failed to register new binary") }
     ) {
       samsungApiClient.registerBinaryFile(
         fileKey = uploadFileResponse.fileKey,
         currentContentInfo = currentContentInfo,
       )
       logger.lifecycle("Successfully registered new binary")
     }
+
+    val hasToSubmitAppValue = hasToSubmitApp.getOrElse("false")
+    logger.lifecycle("Submitting App: $hasToSubmitAppValue")
```

Comment:
Perhaps it's better to be put inside `if` clause and only with simple `Submitting App` message.